operator/v1/types_dns.go

```diff
@@ -376,6 +413,21 @@ const (
 	NetworkResolverType   UpstreamType = "Network"
 )
 
+// ProtocolStrategy is a preference for the protocol to use for DNS queries.
+// +kubebuilder:validation:Enum:=TCP;""
+type ProtocolStrategy string
+
+var (
+	// ProtocolStrategyDefault specifies no opinion for DNS protocol.
+    // If empty, the default behavior of CoreDNS is used. Currently, this means that CoreDNS uses the protocol of the
```

Comment:
Nit can we be consistent with indentation

Reply:
Sorry - I actually turned off auto format in GoLand because it kept changing one of the `// *` formats to remove the bullet points and missed this.

Done.